src/app/example/authContextTest/page.tsx

```diff
@@ -0,0 +1,29 @@
+"use client";
+import { useUser } from "@clerk/nextjs";
+import { useEffect } from "react";
+
+// import { currentUser } from "@clerk/nextjs";
+import React from "react";
+
+export default function Page() {
+  const { isLoaded, isSignedIn, user } = useUser();
```

Comment:
i think you can access unsafe metadata directly in the user object so we don't need the fetch here